README.md

```diff
@@ -1,596 +1,493 @@
-
-#### Setting up the developer environment
-
-###### Initialize the chain on developer machine
-
-The below command creates a json file (`~/openst-setup/config.json`) with all the needed addresses and other constants. It also starts Geth process for the chain.
-
-This is a quick-start option. You could also choose to do this step manually.
-
-```bash
-node ./node_modules/\@openstfoundation/openst.js/tools/initDevEnv.js ~
-
-
-```
-
-#### Creating an OpenST object
-OpenST.js is a thin layer over web3.js. Its constructor arguments are same as that of web3.js.
+This code was tested with geth version: 1.7.3-stable. Other higher versions should also work.
 
+## Creating a OpenST object
+The OpenST object is an entry point: using the OpenST object, a staking can be initiated.
 
 ```js
-// Creating web3 object using the geth endpoint
-const web3Provider = 'http://127.0.0.1:8545';
-
-// Creating OpenST object
-const OpenST = require('./index.js');
-let openST = new OpenST( web3Provider );
-```
+// Creating OpenST.js object
+const OpenST = require('@openstfoundation/openst.js');
 
-OpenST.js also provides access to the web3.js object it creates and uses.
-```js
-// fetch the web3 object to add to the wallet
-let web3 = openST.web3();
 ```
 
-###### Sample constants
-Please use below code if you have set up development environment using init-dev-env.
+## ABI and BIN provider
 
-```js
-const os = require('os');
-let configFilePath = os.homedir() + '/openst-setup/config.json';
-let devEnvConfig = require(configFilePath);
+openst.js comes with an abi-bin provider for managing abi(s) and bin(s).
 
-// Deployer address
-let deployerAddress = devEnvConfig.deployerAddress;
+The abiBinProvider provides abi(s) and bin(s) for the following contracts:
 
-// organization address
-let organizationAddress = devEnvConfig.organizationAddress;
+* [TokenHolder](https://github.com/OpenSTFoundation/openst-contracts/blob/0.10.0-alpha.1/contracts/token/TokenHolder.sol) (TokenHolder contract deployed on UtilityChain)
+* [TokenRules](https://github.com/OpenSTFoundation/openst-contracts/blob/0.10.0-alpha.1/contracts/token/TokenRules.sol) (TokenRules contract deployed on UtilityChain)
+* [PricerRule](https://github.com/OpenSTFoundation/openst-contracts/blob/0.10.0-alpha.1/contracts/rules/PricerRule.sol) (PricerRule is a rule contract deployed on UtilityChain)
+* [GnosisSafe](https://github.com/gnosis/safe-contracts/blob/v0.1.0/contracts/GnosisSafe.sol) (MultiSignature wallet with support for confirmations using signed messages)
+* [DelayedRecoveryModule](https://github.com/OpenSTFoundation/openst-contracts/blob/0.10.0-alpha.1/contracts/gnosis_safe_modules/DelayedRecoveryModule.sol) (Allows to replace an owner without Safe confirmations) 
+* [CreateAndAddModules](https://github.com/gnosis/safe-contracts/blob/v0.1.0/contracts/libraries/CreateAndAddModules.sol) (Allows to create and add multiple module in one transaction)
```

Comment:
`CreateAndAddModules` this probably does not need to be highlighted to the end-user

Reply:
can remain here in the readme for now; but the user reading will definitely get confused.  Perhaps the `## ABI and BIN provider` can move down in the README